cmd/seed-controller-manager/main.go

```diff
@@ -191,7 +191,11 @@ Please install the VerticalPodAutoscaler according to the documentation: https:/
 		// Setup the validation admission handler for kubermatic Cluster CRDs
 		clustervalidation.NewAdmissionHandler(options.featureGates).SetupWebhookWithManager(mgr)
 		// Setup the mutation admission handler for kubermatic Cluster CRDs
-		clustermutation.NewAdmissionHandler(defaultComponentSettings(ctrlCtx)).SetupWebhookWithManager(mgr)
+		seed, err := ctrlCtx.seedGetter()
+		if err != nil {
+			log.Panicf("could not get seed resource: %v", err)
```

Comment:
`Fatalw` instead of `Panicf`.

Reply:
Using panic ensures that defered calls are executed whereas using fatal does not. Do you think we don't need this?

Reply:
Though I can see that most of the places use Fatal. For the consistency this could be changed.